plenum/test/node_catchup/test_node_catchup_after_disconnect.py

```diff
@@ -14,6 +16,7 @@
 
 # TODO: Refactor tests to minimize module-scoped fixtures.They make tests
 # depend on each other
+@pytest.mark.skip(reason="INDY-1297. Node does not catch up on reconnection anymore.")
```

Comment:
Well, actually I don't have confidence that this is a good change to not do catchup for disconnected nodes...
Maybe we can track if we are disconnected from >n-f nodes, and then, we get connected to >n-f back, start catchup explicitly...